net-utils/src/lib.rs

```diff
@@ -421,6 +426,10 @@ fn udp_socket_with_config(config: SocketConfig) -> io::Result<Socket> {
 
     let sock = Socket::new(Domain::IPV4, Type::DGRAM, None)?;
 
+    // Set recv and send buffer sizes to 128MB
+    sock.set_recv_buffer_size(DEFAULT_RECV_BUFFER_SIZE)?;
+    sock.set_send_buffer_size(DEFAULT_SEND_BUFFER_SIZE)?;
```

Comment:
did we confirm that this method directly calls `setsockopt` under the hood?

Reply:
yes it does: https://docs.rs/socket2/0.5.7/src/socket2/socket.rs.html#1015-1024